lib/membrane/rtp/session_bin.ex

```diff
@@ -635,7 +635,8 @@ defmodule Membrane.RTP.SessionBin do
   end
 
   @impl true
-  def handle_notification({:vad, _val} = msg, _from, _ctx, state) do
+  def handle_notification({:vad, value}, {:vad, ssrc}, _ctx, state) do
+    msg = {:vad, {ssrc, value}}
```

Comment:
This generates a new notification that is not documented anywhere. I think you should rather modify the message generated from `RTP.VAD`, you can get the ssrc from the buffer metadata

Reply:
Actually, it turns out that RTC Engine in fact doesn't need to have a knowledge of Active Speaker in the context of the peer, the context of a single track is sufficient. Therefore, we might send an event instead of this notification.